jhipster-jdl.jdl

```diff
@@ -76,34 +76,93 @@ entity CompanionDiagnosticDevice {
     manufacturer String
 }
 
-entity PremarketApproval {
-	device String
-    genericName String
-    applicant TextBlob
+enum FdaSubmissionType {
+	PMA, DE_NOVO, HDE, FIVE_TEN_K
+}
+
+entity FdaSubmission {
```

Comment:
We still need to add the specimen type. But not sure whether it should be in FdaSubmission or CompanionDiagnosticDevice. Waiting for confirmation from Ritika.